### ISSUE TITLE
Unable to connect HA & Deye 5kw Inverter with Stick logger model LSW-3-C

### ISSUE THREAD

**Issue by Kobus4ster** [StephanJoubert/home_assistant_solarman#267]
I am trying to use the Solarman integration to Deye 5kw Inverter with Stick logger model LSW-3-C
![image](https://user-images.githubusercontent.com/130843727/232225764-8cf35006-516f-42f8-839f-d292afff98b1.png)
From below "Unavailable" on all Sensors, it seems that the connection cannot be made to the IP (in the above)
![image](https://user-images.githubusercontent.com/130843727/232225711-b99113cc-a5b6-401b-b522-d59f255aad7e.png)
I have set the configuration as follows 
![Screenshot 2023-04-15 151253](https://user-images.githubusercontent.com/130843727/232283033-e8dcd1ea-f0ef-4668-9fb2-72e43b5ca39a.png)
I added the the sensor in the configuration.yaml as
![Screenshot 2023-04-15 151623](https://user-images.githubusercontent.com/130843727/232283049-771125c6-b7f3-495e-83bc-b32f13bc439e.png)

I have two topics that expect can be the source of my problem:
1) Since the serial number 27... is not in the V5-range, this could be a potential problem, but not sure what exactly need to change, should I need V4 instead.

2) I also have an issue to access the Logger IP 192.168.0.100 from my browser (I tried Brave, Chrome & Edge) - all give the same error: "This site can’t be reached192.168.0.100 refused to connect.". I can see the IP in my DHCP listing and I can also PING the IP with no problem. Trying to resolve this issue, I tried:
- Firewall settings (and even switched it off)
- Antivirus settings (and also set it off)
- tries another PC on my network
but the problem remain

PS: I was able to open the logger when I connected directly to the loggers AP_* WiFi network that it created. I am not sure what the "Remote server B = not connected" means ?
![Screenshot 2023-04-13 200948](https://user-images.githubusercontent.com/130843727/232283195-4c9810ca-4b6d-478d-b8c2-131eb900546c.png)

**Comment by denveronly** [comment]
try connecting to 192.168.0.102 that is your logger IP that is seen on the last screenshot
i have the same V5 with 27xxxxx
to make the connection i used 0.0.0.0 as an ip in the HA integration section and device id is 0
i got it connected but it works not as good as expected.
Battery dischage for example worked for a day registering perfectly, next day i got it not working with last message 10 hours ago (battery was discharging during the daytime)

**Comment by Kobus4ster** [comment]
Thanks for you help @denveronly - unfortunately I can also not access or PING the IP 192.168.0.102.
I have also tried the 0.0.0.0 IP and device id is 0 in the HA, but still get the "unavailable" response. It's really like the logger is not available on my network, but it's IP IP 192.168.0.100 is in the DHCP list on my network. You gave me hope that the connection is possible. Thanks

**Comment by denveronly** [comment]
> Thanks for you help @denveronly - unfortunately I can also not access or PING the IP 192.168.0.102.
> I have also tried the 0.0.0.0 IP and device id is 0 in the HA, but still get the "unavailable" response. It's really like the logger is not available on my network, but it's IP IP 192.168.0.100 is in the DHCP list on my network. You gave me hope that the connection is possible. 

I think that it is not the stick issue, it is your network problem the ip .100 or .102 is already registered by other device
Maybe try specifying in Stick settings not a fhcp connection, but a static ip with
192.168.0.1 router
192.168.0.254 or 253 which absolutely not taken, please scan as Stick ip
255.255.255.0 as subnet mask

after apllying settings try scan one more time if the device had appeared on network
Inverter sticks are too dumb to fail to connect to network

**Comment by Kobus4ster** [comment]
> 
Yes! it's working now - Thanks @denveronly for your advice. I changed the network settings as you suggested and then I could see the logger on my home network with the static IP. I plugged the IP into the HA configuration.yaml file. However, I had to delete and add the Solarman add-in again. Then the Solarman "device" in HA connected and started to show the values from the logger. After about an hour I started to see data on the Energy dashboard. Thanks a mil for your assistance.
